lib/AST/StmtPrinter.cpp

```diff
@@ -1905,8 +1905,8 @@ void StmtPrinter::VisitNullaryBoundsExpr(NullaryBoundsExpr *Node) {
     case BoundsExpr::Any:
       OS << "bounds(any)";
       break;
-    case BoundsExpr::None:
-      OS << "bounds(none)";
+    case BoundsExpr::Unknown:
+      OS << "bounds(Unknown)";
```

Comment:
You probably want this to be `bounds(unknown)` (see case) if it's to match the parsed expression

Reply:
Yes, I've fixed this.